ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/identity_provider_config_service.proto

```diff
@@ -0,0 +1,142 @@
+//
+// The fields of request messages (and sub-messages) are marked either as ``Optional`` or ``Required``:
+// (1) ``Optional`` denoting the client may leave the field unset when sending a request.
+// (2) ``Required`` denoting the client must set the field to a non-default value when sending a request.
+//
+// An identity provider config resource is described by the ``IdentityProviderConfig`` message,
+// An identity provider config resource, once it has been created, can be modified.
+// In order to update the properties represented by the ``IdentityProviderConfig`` message use the ``UpdateIdentityProviderConfig`` RPC.
+// The only fields that can be modified are those marked as ``Modifiable``.
+service IdentityProviderConfigService {
+
+  // Create a new identity provider configuration.
+  // The request will fail if the maximum allowed number of separate configurations is reached.
+  rpc CreateIdentityProviderConfig (CreateIdentityProviderConfigRequest) returns (CreateIdentityProviderConfigResponse);
+
+  // Get the identity provider configuration data by id.
+  rpc GetIdentityProviderConfig (GetIdentityProviderConfigRequest) returns (GetIdentityProviderConfigResponse);
+
+  // Update selected modifiable attribute of a identity provider config resource described
+  // by the ``IdentityProviderConfig`` message.
+  rpc UpdateIdentityProviderConfig (UpdateIdentityProviderConfigRequest) returns (UpdateIdentityProviderConfigResponse);
+
+  // List all existing identity provider configurations.
+  rpc ListIdentityProviderConfigs (ListIdentityProviderConfigsRequest) returns (ListIdentityProviderConfigsResponse);
+
+  // Delete an existing identity provider configuration.
+  rpc DeleteIdentityProviderConfig (DeleteIdentityProviderConfigRequest) returns (DeleteIdentityProviderConfigResponse);
+
+}
+
+message IdentityProviderConfig {
+  // The identity provider identifier
+  // Must be a valid LedgerString (as describe in ``value.proto``).
+  // Required
+  string identity_provider_id = 1;
+
+  // When set, the callers using JWT tokens issued by this identity provider are denied all access
+  // to the Ledger API. Otherwise, the `sub` and `iss` claims provided in the JWT tokens will be 
+  // used to identify the users in the user management database and to grant access accordingly.
+  // Optional,
+  // Modifiable
+  bool is_deactivated = 2;
+
+  // Specifies the issuer of the JWT token that should be used in the "iss" claim of a JWT token to instruct the
+  // participant node to use this identity provider to authenticate the token.
+  // The issuer value is a case sensitive URL using the https scheme that contains scheme, host,
+  // and optionally, port number and path components and no query or fragment components.
+  // Required
+  // Modifiable
+  string issuer = 3;
+
+  // The JWKS (JSON Web Key Set) URL.
+  // The Ledger API will expect all tokens to be signed with RS256 (RSA Signature with SHA-256) with the public
```

Comment:
`Used to obtain a public key which is then used to verify the signatures of JWT access tokens during Ledger API request authentication`
?

Reply:
I'm not sure about using future tense (`will`). We're describing the current behavior of the LAPI server (provided that certain condition are met) so I think present tense is more appropriate.